tests/e2e/e2e_test.go

```diff
@@ -1121,10 +1121,6 @@ func (s *IntegrationTestSuite) TestGeometricTWAP() {
 // This test is to be re-enabled for upgrade once the upgrade handler logic is added and
 // the balancer pool genesis is backported to v14.
 func (s *IntegrationTestSuite) TestStridePoolMigration() {
-	if !s.skipUpgrade {
```

Comment:
Why did we remove this logic again?

Reply:
We disabled this previously because the v14 genesis was not updated. As a result, this test would fail when run with upgrade testing enabled.

Now, that we updated the v14 genesis generation via the `osmolabs/osmosis-e2e-init-chain` script, this test can be enabled with the upgrade